src/main/java/org/alliancegenome/curation_api/services/DataProviderService.java

```diff
@@ -54,15 +54,19 @@ public DataProvider createAffiliatedModDataProvider() {
 	}
 	
 	@Transactional
-	public DataProvider createAllianceDataProvider() {
-		SearchResponse<Organization> orgResponse = organizationDAO.findByField("abbreviation", "Alliance");
+	public DataProvider createOrganizationDataProvider(String dataType) {
+		SearchResponse<Organization> orgResponse = organizationDAO.findByField("abbreviation", dataType);
```

Comment:
Also instead of calling it `dataType` can we call it `organization`?

Reply:
Plus couldn't this be set on the Biological Entity at load time as in the processor, on the Biological Entity DTO and then it doesn't have to be passed around everywhere?